### DISCUSSION TITLE
backward/forward cache

### DISCUSSION THREAD

**Discussion by memchr** [qutebrowser/qutebrowser#8384] [Q&A] [answered]
Does qutebrowser support bfcache? It stores page snapshots to speed up paging. This is a Chromium feature, but it doesn't seem to work with qutebrowser. Are there any knobs that need to be toggled to make this work, or is it not supported?

**Comment by The-Compiler** [comment, marked as answer]
Unsupported by QtWebEngine from what it looks like. I opened an issue: [[QTBUG-131324] Support back/forward cache (bfcache) - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-131324)
